app/models/patient.rb

```diff
@@ -713,7 +713,7 @@ def send_enrollment_notification
   # will be sent immediately without any consideration of the monitoree's preferred_contact_time.
   def send_assessment(send_now: false)
     # Stop execution if in CI
-    return if Rails.env.test?
+    # return if Rails.env.test?
```

Comment:
What was the original reasoning for adding this line in?

Reply:
@holmesie @Bialogs @kylietmo I'd like to remove this line unless there is an unknown reason it is needed here. Are we doing anything like this for other methods that jobs call?

Reply:
I had looked at this before and was confused why that was there. Good to remove it IMO.

Reply:
removing line entirely